nautobot/extras/forms/forms.py

```diff
@@ -259,7 +259,7 @@ class Meta:
 
 class ConfigContextFilterForm(BootstrapMixin, forms.Form):
     q = forms.CharField(required=False, label="Search")
-    # FIXME(glenn) filtering by owner_content_type
+    # 2.0 TODO(glenn) filtering by owner_content_type should be solved by dynamic filter form generation
```

Comment:
If so then this should be something to test as part of, or following, #2348, rather than deferred to 2.0 to address.